src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -39,9 +39,10 @@
 #include <viam/sdk/config/resource.hpp>
 #include <viam/sdk/module/service.hpp>
 #include <viam/sdk/registry/registry.hpp>
+#include <viam/sdk/resource/reconfigurable.hpp>
```

Comment:
Changes in this file are all to catch up with changes in the C++ SDK. There are no material changes to how the module functions.